engine/src/main/java/org/terasology/rendering/dag/nodes/WorldReflectionNode.java

```diff
@@ -69,17 +66,16 @@ public void initialise() {
         this.renderingConfig = config.getRendering();
         this.chunkShader = worldRenderer.getMaterial("engine:prog.chunk");
         this.playerCamera = worldRenderer.getActiveCamera();
-        addDesiredStateChange(new BindFBO(SCENE_REFLECTED_FBO, frameBuffersManager));
-        addDesiredStateChange(new SetViewportSizeOf(SCENE_REFLECTED_FBO, frameBuffersManager));
-
+        addDesiredStateChange(new BindFBO(SCENE_REFLECTED_FBO));
+        addDesiredStateChange(new SetViewportSizeOf(SCENE_REFLECTED_FBO));
+        addDesiredStateChange(new SetFacesToCull(GL_FRONT));
```

Comment:
Careful. This imply face culling must be enabled. We need an additional state change in this list.